DataConnectors/AWS-S3/Utils/CommonAwsPolicies.ps1

```diff
@@ -122,33 +122,64 @@ param (
 	[Parameter(position=2)]
 	[ValidateNotNullOrEmpty()]
 	[string]
-	$SqsArn
+	$SqsArn,
+	[Parameter()]
+	[bool]
+	$IsCustomLog
 )  
 
-	$sqsEventConfig = "
-   {
-	   'QueueConfigurations': [
-			{
-			  'Id':'$EventNotificationName',
-			  'QueueArn': '$SqsArn',
-			  'Events': ['s3:ObjectCreated:*'],
-			  'Filter': {
-				'Key': {
-				  'FilterRules': [
-					{
-					  'Name': 'prefix',
-					  'Value': '$EventNotificationPrefix'
-					},
-					{
-					  'Name': 'suffix',
-					  'Value': '.gz'
+	if($true -eq $IsCustomLog)
```

Comment:
both object looks the same, except `FilterRules`.
is it possible to assign the   `FilterRules` to a local member according to `IsCustomLog`, and then use this member in the QueueConfigurations object. 

this way we have less duplictae code, easier to update in the future.